src/util/bit_util.cuh

```diff
@@ -0,0 +1,96 @@
+
+/*
+ * Copyright 2018 BlazingDB, Inc.
+ *     Copyright 2018 Alexander Ocsa <alexander@blazingdb.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+namespace gdf {
+namespace util {
+
+static constexpr int ValidSize = 32;
+using ValidType = uint32_t;
+
+__host__ __device__ __forceinline__
+  size_t
+  valid_size(size_t column_length)
+{
+  const size_t n_ints = (column_length / ValidSize) + ((column_length % ValidSize) ? 1 : 0);
+  return n_ints * sizeof(ValidType);
+}
+
+__host__ __device__ __forceinline__ bool get_bit(const gdf_valid_type* const bits, size_t i)
```

Comment:
Comment out `get_bit` and instead add a reference to the corresponding function in `utils.h`